projects/admin/src/app/service/user.service.ts

```diff
@@ -86,4 +129,17 @@ export class UserService {
       concatAll()
     );
   }
+
+  /**
+   * Check if you are an access ton admin interface
+   */
+  private isAuthorizedAccess() {
+    const adminRoles = this._appConfigService.adminRoles;
+    this._allowInterfaceAccess = this.user.getRoles()
+    .filter((role: string) => {
+      if (adminRoles.indexOf(role) > -1) {
```

Comment:
You can probably use some here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some